src/neuroglancer/layer_panel.ts

```diff
@@ -182,6 +186,32 @@ class LayerWidget extends RefCounted {
       this.panel.layerManager.removeManagedLayer(this.layer);
       event.stopPropagation();
     });
+    const initColor = layer.initialSpecification.annotationColor || '';
+    const colorElement = makeTextIconButton('█', initColor);
```

Comment:
Maybe make the button the same as the color input field in the `AnnotationLayer` - just in case this box doesn't get rendered properly / differently.